data/strings/en.ini

```diff
@@ -55,6 +55,18 @@ Automatic Remap
 ||&OK||&Cancel"
 END
 cannot_delete_all_layers = Error<<You cannot delete all layers.||&OK
+deleting_tilemap_will_delete_tileset = <<<END
+Warning
+<<Deleting '{0}' layer will delete its associated tileset.
+<<Do you want to continue anyway?
+||&OK||&Cancel
+END
+deleting_tilemaps_will_delete_tilesets = <<<END
+Warning
+<<Deleting '{0}' layers will delete their associated tilesets.
+<<Do you want to continue anyway?
+||&OK||&Cancel
+END
```

Comment:
I'm not sure about this alert, probably a good idea to add it. If we keep it, two things:
1. Add this alert as optional (search for `OptionalAlert`)
2. Could we merge these two messages in one (with one generic string for both cases)

Reply:
Sure, will make it optional